src/Gui/CommandWindow.cpp

```diff
@@ -370,7 +370,7 @@ class FilterStatusBar : public QObject
 //    Q_OBJECT
 
 public:
-    FilterStatusBar(Action * action):QObject() {this->action = action;}
+    FilterStatusBar(Action * action):QObject(action) {this->action = action;}

```

Comment:
FilterStatusBar is leaking, I am not sure about this one, but attaching it to the action attaches to the MainWindow and destroys succesfully

Reply:
c4e383829b0